src/cmake/externalpackages.cmake

```diff
@@ -236,12 +244,12 @@ checked_find_package (OpenVDB
                       DEPS         TBB
                       DEFINITIONS  -DUSE_OPENVDB=1)
 
-checked_find_package (Ptex PREFER_CONFIG)
-if (NOT Ptex_FOUND OR NOT Ptex_VERSION)
+checked_find_package (ptex PREFER_CONFIG)
+if (NOT ptex_FOUND OR NOT ptex_VERSION)
```

Comment:
Same. Ptex does seem to have an exported cmake these days, and I don't mind using that and getting rid of our FindPtex.cmake module. But let's understand clearly what that does to the minimum Ptex that we're able to support.

Reply:
Fixed case